core/file/atime_spec.rb

```diff
@@ -15,7 +15,7 @@
     File.atime(@file).should be_kind_of(Time)
   end
 
-  platform_is :linux do
+  guard -> { platform_is :linux or (platform_is :windows and ruby_version_is '2.5') } do
```

Comment:
Does `stat -c%x` work on Windows then?

Reply:
Technically, no.  Does it work in the CI environment, yes.

I believe both test-all & test-spec have *nix utilities here and there.  Since spec requires build tools for the CAPI tests, the build tools are accessible.  One also needs some *nix utilities to build Ruby under vc (mswin).

Reply:
OK, that's good enough then :)